Library/Homebrew/standalone/sorbet.rb

```diff
@@ -1,11 +1,41 @@
 # typed: true
 # frozen_string_literal: true
 
-# Explicitly prevent `sorbet-runtime` from being loaded.
-def gem(name, *)
-  raise Gem::LoadError if name == "sorbet-runtime"
+require "sorbet-runtime"
 
-  super
-end
+# Disable runtime checking unless enabled.
+unless ENV["HOMEBREW_SORBET_RUNTIME"]
+  # @private
+  module TNoChecks
```

Comment:
I don't understand what any of the code in this file is doing and why it's here 😅. Could it get some more comments?

Reply:
> `# Disable runtime checking unless enabled.`

I could maybe add more?

Reply:
Yeh, please, pretty much just this comment but in Ruby 😁